.travis.yml

```diff
@@ -3,7 +3,10 @@ python:
   - 3.4
   - 3.5
   - 3.6
```

Comment:
It seems like the build fails for Python 3.6. Removing the 3.4 and 3.6 should fix this - see @charlienewey's  [commit](https://github.com/soton-data-mining/job-salary-prediction/pull/14/commits/8fab3b2fa784423dbf7618e6046fd8717cd31f76)